test/unit/processing/components/get-schedule-dates.test.js

```diff
@@ -95,79 +95,68 @@ describe('get schedule dates', () => {
 
   test('should insert top up adjustment after last outstanding payment in original schedule', () => {
     const schedule = getScheduleDates(previousPaymentSchedule, newPaymentScheduleTopUp, deltaValue)
-    expect(schedule[2].period).toEqual(moment().format('MMM YYYY'))
+    expect(schedule[0].period).toEqual(moment().format('MMM YYYY'))
   })
 
   test('Adjustment payment Type must be Immediate payment', () => {
     const schedule = getScheduleDates(previousPaymentSchedule, newPaymentScheduleTopUp, deltaValue)
-    expect(schedule[2].paymentType).toEqual(IMMEDIATE)
+    expect(schedule[0].paymentType).toEqual(IMMEDIATE)
   })
 
-  test('should retain previous payment value for paid segments if top up', () => {
+  test('should not retain previous payment value for paid segments if top up', () => {
     const schedule = getScheduleDates(previousPaymentSchedule, newPaymentScheduleTopUp, deltaValue)
-    expect(schedule[0].value).toEqual('250.00')
-    expect(schedule[1].value).toEqual('250.00')
+    expect(schedule[0].paymentType).toEqual(IMMEDIATE)
```

Comment:
Nice, can we add above this `expect(schedule[0].paymentType).not.toEqual(QUATERLY)`

Reply:
Just this missed comment if you agree

Reply:
already in the code file, check:

https://github.com/DEFRA/ffc-pay-statement-constructor/blob/818209_Receiving-Remaining-amount-from-Constructor-for-reduction-schedule/test/unit/processing/components/get-schedule-dates.test.js

line 104

Reply:
Apologises, GH has lied to me!